index.js

```diff
@@ -170,7 +180,11 @@ function fastifyView (fastify, opts, next) {
       if (!this.res.getHeader('content-type')) {
         this.header('Content-Type', 'text/html; charset=' + charset)
       }
-      this.send(toHtml(data))
+      if (options.useHtmlMinifier && (typeof options.useHtmlMinifier.minify === 'function')) {
+        this.send(options.useHtmlMinifier.minify(toHtml(data), options.htmlMinifierOptions || {}))
```

Comment:
I think this should not be needed here, as the minifier is already applied in `readCallback` before putting it in the cache. Can you confirm/check?

Reply:
Will remove and try the unit test again to verify that it is not necessary.

Reply:
Completed